.github/workflows/win.yml

```diff
@@ -292,6 +292,25 @@ jobs:
           cp libs/* crystal/lib/
           cp src/* crystal/src -Recurse
           rm crystal/src/llvm/ext/llvm_ext.obj
+          echo "$(pwd)\crystal" >> $env:GITHUB_PATH
+
+      - name: Get latest shards release
+        run: |
+          echo "SHARDS_RELEASE_TAG=$(Invoke-RestMethod https://api.github.com/repos/crystal-lang/shards/releases/latest | Select-Object -ExpandProperty tag_name)" >> ${env:GITHUB_ENV}
+
+      - name: Download latest shards release
+        uses: actions/checkout@v2
+        with:
+          repository: crystal-lang/shards
+          ref: ${{ env.SHARDS_RELEASE_TAG }}
+          path: shards
+
+      - name: Build latest shards release
+        working-directory: ./shards
+        run: |
+          make
```

Comment:
That... just works? 😲

Reply:
It seems the Windows CI environment just happens to have everything; `make` from Chocolatey, `cat` and `stat` from Git Bash, `find` from Windows. Those `find` commands actually fail:

https://github.com/crystal-lang/shards/blob/1739c0fb96cf16e405bcdebf91357540d87243b3/Makefile#L11

https://github.com/crystal-lang/crystal/runs/4437326774?check_suite_focus=true#step:40:37

Reply:
Also if the Makefile runs under Powershell, it will fail on _Windows_ Powershell because the `||` operator is unavailable in the latter.

Reply:
Hm, although _Windows_ PowerShell is gradually being phased out, this is still a good point. I can skip the make file, pull down molinillo, and build with `crystal build --no-color src/shards.cr`. That way we avoid any messiness related to PowerShell editions or shallow bash aliases.

Reply:
I have never had any problems with `make`. Been running for over a year. I very much rely on it in CI and if something breaks, everyone's builds will break and I'll be the first to deal with that.
https://github.com/crystal-lang/install-crystal/blob/917bac4f2f781f25d0d6f85c6c6bd7426eebdf78/index.js#L265
So I think with `make` this was better.